data-raw/create_sdtms_data.R

```diff
@@ -0,0 +1,142 @@
+#' This script generates R documentation files for datasets in the 'pharmaversesdtm' package.
+#'
+#' It reads metadata from a JSON file ('inst/extdata/sdtms-specs.json') and uses that information to create R documentation files
+#' for each dataset. The script utilizes helper functions to handle metadata retrieval and format the documentation properly.
+#' The generated documentation files are saved in the 'R' directory and should not be manually edited, as they are subject to automatic regeneration.
+
+# Load the necessary library for JSON
+library(jsonlite)
+
+# Load the JSON file with metadata
+specs <- fromJSON("inst/extdata/sdtms-specs.json")$`sdtms-specs`
+
+# Helper function to get attribute for a column, with default as "undocumented field"
+get_attr <- function(data, col_name) {
+  att <- attr(data[[col_name]], "label")
+  if (is.null(att) || att == "null") {
+    att <- "undocumented field"
+  }
+  return(att)
+}
+
+# Helper function to check the length of a string and append " # nolint" if it exceeds 90 characters
+check_and_append_nolint <- function(text) {
+  if (nchar(text) > 80) {
+    text <- paste0(text, " # nolint")
+  }
+  return(text)
+}
```

Comment:
This doesn't work properly because the "# nolint" appears in the rendered documentation:
![image](https://github.com/user-attachments/assets/c0aeb8ed-c806-4331-bb59-0ae1fa1bd57f)